### PR TITLE
No search results response handle

### LINKED ISSUE #1894: Handle API response when no results with search.
In files `/Drive` page, search for a random string that has no results. 

The App crashes.

### PR DESCRIPTION
**Pull request by tanmoyAtb**
closes #1894

### CODE REVIEW

**Comment by asnaith** [comment]
Nice fix, I'm no longer seeing the crash. 👍

 I am seeing duplicate toasts though, is this related or should I create a separate issue for this? 

https://user-images.githubusercontent.com/3982665/151848615-7352370d-58a8-4f6c-a469-8d13e81728fe.mov

**Review by FSM1 — APPROVED** [review]

**Comment by FSM1** [comment]
> I am seeing duplicate toasts though, is this related or should I create a separate issue for this?

Yeah also just noticed that. At first I thought it was purely the two consecutive calls failing, but turns out the error is thrown because of a empty file system.

**Comment by tanmoyAtb** [comment]
> Nice fix, I'm no longer seeing the crash. 👍
> 
> I am seeing duplicate toasts though, is this related or should I create a separate issue for this?
> 
>  Screen.Recording.2022-01-31.at.10.04.33.AM.mov

Good find. I'll check into this as well.

**Comment by Tbaut** [comment]
> TypeError: null has no properties

That's the error in the console

**Comment by tanmoyAtb** [comment]
> > TypeError: null has no properties
> 
> That's the error in the console

Updates are in.

**Review by FSM1 — APPROVED** [review]

File: packages/files-ui/src/Components/Modules/FileBrowsers/SearchFileBrowser.tsx [hunk 1]
```diff
@@ -63,8 +55,7 @@ const SearchFileBrowser: React.FC<IFileBrowserModuleProps> = ({ controls = false
       }
     }
     onSearch()
-    // eslint-disable-next-line
-  }, [searchTerm])
+  }, [searchTerm, getSearchResults])
```
Comment (Tbaut) [inline packages/files-ui/src/Components/Modules/FileBrowsers/SearchFileBrowser.tsx:58]: :heart:

**Review by Tbaut — APPROVED** [review]
awesome

**Pull request merged**